prowler/providers/aws/services/eventbridge/eventbridge_bus_exposed/eventbridge_bus_exposed.py

```diff
@@ -0,0 +1,48 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.eventbridge.eventbridge_client import (
+    eventbridge_client,
+)
+
+
+class eventbridge_bus_exposed(Check):
+    def execute(self):
+        findings = []
+        for bus in eventbridge_client.buses.values():
+            report = Check_Report_AWS(self.metadata())
+            report.status = "PASS"
+            report.status_extended = (
+                f"EventBridge event bus {bus.name} is not exposed to everyone."
+            )
+            report.resource_id = bus.name
+            report.resource_arn = bus.arn
+            report.resource_tags = bus.tags
+            report.region = bus.region
+            # If the "Principal" element value is set to { "AWS": "*" } and the policy statement is not using any Condition clauses to filter the access, the selected AWS EventBridge bus is publicly accessible.
+            if bus.policy and "Statement" in bus.policy:
+                for statement in bus.policy["Statement"]:
```

Comment:
The same again, move this into a function please.

Reply:
Done